### ISSUE TITLE
Add tests cases in blktests

### ISSUE THREAD

**Issue by old-memories** [ming1/ubdsrv#50]
Hi Ming,

Shall we add some ublk tests in blktests? So it could be more convenient for testing. For example, our test group can easily run blktests and report test failure. But deploying ublksrv and built-in tests may be a little bit harder.

I think the work is simple because in blktests, there is a miniublk and we can move some useful tests in ublksrv into blktests. At least we can test ublk add/del in blktests I think.

**Comment by ming1** [comment]
Yeah, totally agree, the miniublk is supposed to cover ublk driver regression test.

**Comment by old-memories** [comment]
Ming. Would you mind if I do this work now? We are going to release a new kernel version and ublk is included.

**Comment by ming1** [comment]
Sure, please feel free to add ublk tests on blktests.
